setup.py

```diff
@@ -48,8 +49,8 @@
     install_requires=[
         "Wagtail>=4.1",
         "user-agents>=2.2,<2.3",
-        "numpy>=1.19.4,<1.25",
-        "scipy>=1.5.4,<1.10",
+        "numpy>=1.19.4,<1.27",
+        "scipy>=1.5.4,<1.12",
```

Comment:
Maybe we can open this up and make this `<2`

(Would be even more awesome if we could move the entire 'winner' calculation which requires these dependencies to the database and drop the need to install these heavyweight dependencies. Maybe some future day...)

Reply:
> Maybe we can open this up and make this `<2`

Since we're using a tiny bit of the numpy/scipy APIs, this does seem like a low-risk change. I've loosened the constraints.